riff-raff/app/assets/javascripts/form-autocomplete.coffee

```diff
@@ -3,6 +3,8 @@ menuOpen = false
 
 updateBuildInfo = (buildNumber) ->
   $('#build-info').load(jsRoutes.controllers.DeployController.buildInfo(selectedProject, buildNumber).url)
+  showProdBranchCheckboxAndEnDisableDeployButtons(false)
```

Comment:
I think there is a small naming issue here - should this be 'showProdBranchCheckboxAndDisableDeployButtons' instead of 'showProdBranchCheckboxAnd**En**DisableDeployButtons'?